test/functional/rpc_net.py

```diff
@@ -163,6 +163,16 @@ def _test_getpeerinfo(self):
 
         # Check dynamically generated networks list in getpeerinfo help output.
         assert "(ipv4, ipv6, onion, i2p, not_publicly_routable)" in self.nodes[0].help("getpeerinfo")
+        # This part is slightly different comparing to the Bitcoin implementation. This is expected because we create connections on network setup a bit differently too.
+        # We also create more connection during the test itself to test mn specific stats
+        assert_equal(peer_info[0][0]['connection_type'], 'manual')
+        assert_equal(peer_info[0][1]['connection_type'], 'inbound')
+
+        assert_equal(peer_info[1][0]['connection_type'], 'inbound')
+        assert_equal(peer_info[1][1]['connection_type'], 'manual')
+
+        assert_equal(peer_info[1][2]['connection_type'], 'manual')
```

Comment:
19725: nit: should group them by node
```suggestion
        assert_equal(peer_info[1][1]['connection_type'], 'manual')
        assert_equal(peer_info[1][2]['connection_type'], 'manual')

```

Reply:
updated grouping , requesting review @UdjinM6 @knst